src/main/java/org/ibp/api/domain/common/PagedResult.java

```diff
@@ -27,14 +30,14 @@
 	private String sortBy;
 	private String sortOrder;
 
-	public static final int MAX_PAGE_SIZE = 10000;
+	public static int MAX_PAGE_SIZE;
```

Comment:
I think we can convert MAX_PAGE_SIZE to a non-static local variable such as:
`protected int maxPageSize;`

Reply:
The instances of PagedResult and BrapiPagedResult (inheriting PagedResult) are being instantiated via new, if we convert the variable MAX_PAGE_SIZE to non-static the value set during spring instantiate will be ignored. Please refer to this [link](https://mkyong.com/spring/spring-inject-a-value-into-static-variables/) for injecting value into static variable